spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+describe ReviewsController do 
+  describe "POST create" do
+    let(:video) {Fabricate(:video)}
```

Comment:
I would add a blank line here for extra separation. All bunched up this is hard to read.